src/pages/questionPage.js

```diff
@@ -23,13 +23,25 @@ export const initQuestionPage = (userInterface) => {
     .addEventListener('click', nextQuestion)
 };
 // check answers if correct or not 
+let isSelectedOneAnswer= false;
 const getAnswer = (e)=> {
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+if(!isSelectedOneAnswer){
   const answer = e.target.innerText;
   currentQuestion.selected=answer.charAt(0)
-  currentQuestion.selected===currentQuestion.correct ? console.log('selected correct answer') : console.log('selected wrong answer')
+  showAnswerIsCorrect(currentQuestion.selected, currentQuestion.correct,e.target)
```

Comment:
You don't need to pass `currentQuestion.selected` and `currentQuestion.correct`, you can just pass `currentQuestion.selected === currentQuestion.correct`, or create a variable called eg `isAnswerCorrect` and store there `currentQuestion.selected === currentQuestion.correct`, and then pass that variable as argument to `showAnswerIsCorrect`